pkg/resourcetotelemetry/resource_to_telemetry.go

```diff
@@ -32,15 +35,22 @@ import (
 //	}
 type Settings struct {
 	// Enabled indicates whether to convert resource attributes to telemetry attributes. Default is `false`.
-	Enabled bool `mapstructure:"enabled"`
+	Enabled bool            `mapstructure:"enabled"`
+	Exclude MatchAttributes `mapstructure:"exclude"`
```

Comment:
Could you document these new fields and structs? Feel free to use the one on other components as a start.

Reply:
done